core/src/main/resources/schema/h2/h2-create-schema.sql

```diff
@@ -18,12 +18,14 @@ CREATE TABLE IF NOT EXISTS "event_journal" (
 CREATE UNIQUE INDEX "event_journal_ordering_idx" on "event_journal" ("ordering");
 
 CREATE TABLE IF NOT EXISTS "event_tag" (
-    "event_id" BIGINT NOT NULL,
+    "event_id" BIGINT,
```

Comment:
It would be better if new users just don't have this column.

I know, legacy mode is enabled by default and if you just start today with the plugin you will be using it in legacy mode without even noticing it. 

I don't have a solution. I'm just mentioning it so we can think together about alternatives. 

In the worst case scenario, we leave it as is and people will have a column that is never used.

Reply:
Probably, we need to go with that and on a next release we simply drop this column and we remove the legacy mode flag. 

Then users will first need to move to the previous version, run the migrate and then move to the next version. After all that they will be able to drop the `event_id` column.